Add-ons/UmbracoForms/Developer/Extending/Adding-a-Fieldtype.md

```diff
@@ -64,6 +64,21 @@ The view takes care of generating the UI control and setting its value.
 
 On the view, it is important to note that the ID attribute is fetched from `@Model.Id`. You'll also see that we are using jQuery validate unobtrusive to perform client side validation so that's why we are adding the data* attributes.
 
+## Partial view for theme
```

Comment:
I think it would make sense to make this a sub heading to the "Partial Views section above - what do you think? 🤔

```suggestion
### Partial view for themes
```